cache/test_conditional.py

```diff
@@ -147,19 +164,6 @@ def test_none_match_HEAD(self):
             method="HEAD",
         )
 
-    def test_none_match_POST(self):
```

Comment:
Why do you remove this test? Maybe we can change it to a negative test? or add `unitest.expectedFailure` decorator?